django_loci/static/django-loci/css/loci.css

```diff
@@ -4,7 +4,6 @@
 /* hide redundant heading in objectlocationinline */
 .loci.coords > h2{ display: none }
 /* improve raw_id_field look */
-.coords .field-location input[type=text]{ display: none }
```

Comment:
I am not sure why this css is used. Can you please confirm removing this is not affecting anything.